packages/angular_devkit/core/node/host.ts

```diff
@@ -224,24 +223,30 @@ export class NodeJsSyncHost implements virtualFs.Host<fs.Stats> {
   }
 
   write(path: Path, content: virtualFs.FileBuffer): Observable<void> {
-    // Create folders if necessary.
-    const _createDir = (path: Path) => {
-      if (fs.existsSync(getSystemPath(path))) {
-        return;
-      }
+    return new Observable(obs => {
+      // Create folders if necessary.
+      const _createDir = (path: Path) => {
+        if (fs.existsSync(getSystemPath(path))) {
```

Comment:
Probably a refactor for the future but for all the sync calls could we use the async versions with `utils.promisify` and convert and return the _observabled_ promise?

Reply:
You mean the async ones? The sync ones need to be sync.

Reply:
Yes. Sorry for the confusion.